package.json

```diff
@@ -9,7 +9,8 @@
   ],
   "scripts": {
     "setup": "yarn && wsrun -s -c install",
-    "dev": "wsrun -p @gamedao-haiku/app -c dev",
+    "dev": "yarn build:graphql && yarn dev:app",
```

Comment:
Will not work on all systems. Use` yarn npm-run-all -s build:graphql dev:app`

Reply:
🤷  then either use wsrun consistently or npm run all...

Reply:
i also have not seen a system where running two commands in a sequence failed to work, why should it?

Reply:
> 🤷 then either use wsrun consistently or npm run all...
Feel free to change it. 

> i also have not seen a system where running two commands in a sequence failed to work, why should it?
Because the operators do not do the same on every system, that's why you should avoid them.
You haven't tried a Windows console for a long time or ?

Reply:
just stop using windows ^^